Dockerfile

```diff
@@ -1,23 +1,9 @@
-FROM golang:1.13-alpine as builder
```

Comment:
I understand the removal of this section due to the build script but I like the idea of keeping this part in there for other folks to have a nice way of building locally without having to run the full `buildDocker.sh` script.

Reply:
This new version also requires you to build the binaries ahead of the Docker file which works fine for the pipeline but now forces people into a 2-step process when building locally.